scripts/aad-app-reg.sh

```diff
@@ -3,7 +3,7 @@
 # Setup Script
 set -euo pipefail
 
-function usage()
+function show_usage()
```

Comment:
minor: you've changed name to `show_usage`. This is a better name, but is now inconsistent with other scripts (e.g. publish_register_bundle.sh also in this PR).

Meh....feel free to ignore.